README.md

```diff
@@ -72,19 +72,21 @@ This component wraps your entire app (but within the DrizzleProvider) and will s
 
 `labels` (array) Custom labels; will follow ABI input ordering. Useful for friendlier names. For example "\_to" becoming "Recipient Address".
 
+`render` (function with two arguments) Render property to overwrite form generation. The first argument is an object the maps all method argument names to [react refs](https://reactjs.org/docs/refs-and-the-dom.html). Your form input fields require a `ref` property. Note: [If you use the Material-UI library, the element needs an additional `onChange` property](https://stackoverflow.com/questions/51310609/unable-to-retrieve-the-input-field-of-material-ui-using-refs-in-react-js/55195196#55195196). The second argument is the instant of the `ContractForm`, that means the `ContractForm`s `this`. Use that for calling `ContractForm.setSendArgs` or `ContractForm.handleSubmit` from your render property.
```

Comment:
This is a bit hard to read, can you split it up into multiple paragraphs?

Reply:
ok.